.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I believed that it would be good to have a debugger set up from the get go. I know it is vscode specific but can still be useful. What do you think? From what I know it should work across devices.